tests/parsers/biorad_bioplex_manager/testdata/bio-rad_bio-plex_manager_example_01.json

```diff
@@ -256,9 +256,6 @@
                                         }
                                     }
                                 ]
-                            },
```

Comment:
Note to reviewer: this was a harmless bug in the existing implementation, should not include empty list